docs/source/workflows/generative_design.rst

```diff
@@ -1,4 +1,4 @@
-.. generative_design_execution:
+.. _generative_design:
```

Comment:
It felt weird that this file was given its own special section outside of the AI Engine documentation, so I am moving it within that heading.